contribute/deprecation-policy.md

```diff
@@ -0,0 +1,34 @@
+# Deprecation policy
+
+Prior to Grafana 10, major releases were an opportunity to make breaking changes and remove code we didn’t want to maintain.
+
+Going forward we will do our best to **not** make breaking changes, in order to make upgrading Grafana as easy and reliable as possible. From now on we will avoid making unexpected breaking changes to the customer experience and our ability to operate our service.
```

Comment:
This has always from day 1 been a major goal and focus (to make upgrading as easy and as pain-free as possible). So not sure what is changing (other than adding a formal policy). This seems to conflate unexpected breaking changes and deprecating features. 

I would rephrase this a bit. 

```suggestion
Major releases are when we will be making breaking changes, like changing behavior or removing a feature. 

We always do our best **not** to introduce breaking changes in order to make upgrading Grafana as easy and reliable as possible. However, at times we have to introduce a breaking change by changing behavior or by removing a feature.  

But in order to remove a feature and introduce a breaking change we need to create a deprecation plan. 
```